core/bind/core_bind.cpp

```diff
@@ -2462,7 +2462,7 @@ void _Directory::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_drive", "idx"), &_Directory::get_drive);
 	ClassDB::bind_method(D_METHOD("get_current_drive"), &_Directory::get_current_drive);
 	ClassDB::bind_method(D_METHOD("change_dir", "todir"), &_Directory::change_dir);
-	ClassDB::bind_method(D_METHOD("get_current_dir"), &_Directory::get_current_dir);
+	ClassDB::bind_method(D_METHOD("get_current_dir", "include_drive"), &_Directory::get_current_dir, DEFVAL(true));
```

Comment:
I wonder if this doesn't break compat for the GDNative API?

Reply:
I don't think so because it's not a core core class (repetition intended) and the parameter is optional. However, to be honest, I'm in doubt. I need some help to confirm.

Reply:
Truth is it breaks it indeed. I'll submit an update.